packages/aws-client/src/S3.ts

```diff
@@ -579,6 +579,16 @@ export const recursivelyDeleteS3Bucket = improveStackTrace(
   }
 );
 
+/**
+* Delete a list of buckets and all of their objects from S3
+*
+* @param {Array} buckets - list of bucket names
+* @returns {Promise} the promised result of `S3.deleteBucket`
+**/
+export const deleteS3Buckets = async (
```

Comment:
😱 I think this is fine as a test helper.... and I guess it belongs here.   We should make sure our 'we care about them' buckets have versioning/deletion protection on though :)

Reply:
Ha yeah, good point. This is a dangerous one. I wanted here as a share util mainly just for tests. We do this a lot in our cleanup steps. Would you sleep easier at night if this was in a test helper somewhere?

Reply:
No, I think this is the right spot for it, just a good reminder to make sure we have good bucket hygiene.